extensions/VisualEditor/wikia/modules/ve/ui/dialogs/ve.ui.WikiaMediaInsertDialog.js

```diff
@@ -151,7 +152,12 @@ ve.ui.WikiaMediaInsertDialog.prototype.initialize = function () {
 
 	this.mainPage = new OO.ui.PageLayout( 'main', { $content: this.$mainPage } );
 	this.searchPage = new OO.ui.PageLayout( 'search', { $content: this.search.$element } );
-	this.pages.addPages( [ this.mainPage, this.searchPage ] );
```

Comment:
It would be great if that approach could work also for hiding what currently you are hiding with class ```ve-ui-wikiaMediaQueryWidget-uploadWrapper-video```